Index.html

```diff
@@ -171,6 +171,79 @@ <h3 class="h3">Skills</h3>
             </ul>
         </section>
         
+        <section id="contact" class="contact">
+            <img src="./Contents/Resources/Pictures/image_geometry_1.svg" alt="" class="corner__geometry">
+            <img src="./Contents/Resources/Pictures/contact_shape_one.svg" alt="" class="contact__image__left">
+            <div class="article__form" >
+                <h2 class="h2 contact__h2">Get Started</h2>
+                <p class="contact__text">I Prefer To Speak By My Gestures</p>
+                <form action="https://formspree.io/f/" method="post" class="contact__form">
```

Comment:
- [ ] Kindly consider adding a valid action link. In other, for you to do that please click on the [link](https://formspree.io/) provided in the project requirement and register an account. After registering an account you will be given a unique link which is what you're going to use in your form action to collect use data. Your unique link should look something like `https://formspree.io/f/{form_id}`. See below for more details: 
<img width="664" alt="Screenshot 2023-05-02 at 19 02 06" src="https://user-images.githubusercontent.com/107192076/235762163-0ef8c419-3c87-46f3-8631-615ac30bfaba.png">